_includes/styles.html

```diff
@@ -1,9 +1,15 @@
 {% assign _styles = '' | split: '' %}
-{% assign _default_styles = '' | split: ''
-  | push: '/assets/css/main.css' %}
+{% assign _site_styles = site.styles %}
+{% unless _site_styles -%}
+  {% assign _uswds_css = '/assets/css/main.css' %}
+  {% assign _site_styles = '' | split: ''
+    | push: _uswds_css %}
+{% endunless %}
+{% assign _theme_key = site.theme_variant %}
+{% assign _theme_styles = site.data.themes[_theme_key].styles %}
 {% assign _styles = _styles
-  | push: _default_styles
```

Comment:
This overwrites some of the changes that were made in https://github.com/18F/uswds-jekyll/pull/119, so I can rework this just to include the pieces that are specific to the theme stylesheets.

Reply:
I will clean this up.